packages/server/src/app/db/content/blogs/blogs.service.ts

```diff
@@ -1,4 +1,87 @@
 import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { JwtPayload } from '@pulp-fiction/models/auth';
+import { PaginateModel, Types } from 'mongoose';
+import { UsersService } from '../../users/users.service';
+
+import { BlogsContentDocument } from './blogs-content.document';
+import { BlogForm, PubChange, PubStatus } from '@pulp-fiction/models/content';
+import { sanitizeHtml, stripAllHtml } from '@pulp-fiction/html_sanitizer';
+import { countPlaintextWords } from '@pulp-fiction/word_counter';
+
 
 @Injectable()
-export class BlogsService {}
+export class BlogsService {
+    constructor(@InjectModel('BlogContent') private readonly blogsModel: PaginateModel<BlogsContentDocument>,
+        private readonly usersService: UsersService) {}
+
+    /**
+     * Creates a new blogpost and saves it to the database. Returns the newly
+     * created blog as a promise.
+     *
+     * @param user The user making the blog.
+     * @param blogInfo The blog's information.
```

Comment:
Missing comment doc for `parentId`. Also, what _is_ a parent, in the context of a blog?